release/OpenWrt/Makefile

```diff
@@ -59,13 +59,13 @@ define Package/$(PKG_NAME)/install
 
 	# install shell script, conf and service
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) ./files/hetzner_ddns.sh $(1)/usr/bin
+	$(INSTALL_BIN) ./files/hetzner_ddns $(1)/usr/bin
 
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/hetzner_ddns.openwrt.rc $(1)/etc/init.d/hetzner_ddns
 
 	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/hetzner_ddns.conf $(1)/etc/config/hetzner_ddns.conf
+	$(INSTALL_DATA) ./files/hetzner_ddns.conf $(1)/etc/config/hetzner_ddns.conf
```

Comment:
Ah, is it better to do it with `$(INSTALL_DATA)` vs. `$(INSTALL_CONF)`?

Reply:
Yes, as mentioned in https://github.com/filiparag/hetzner_ddns/pull/22/commits/ec11febc74acc63381d3aa7ad6d61149eaf5450e.

According to https://openwrt.org/docs/guide-developer/packages#file_installation_macros 

INSTALL_CONF is short for install -m600

while

INSTALL_DATA is short for install -m644

This should give read access to all users